test/unit/deployment.bats

```diff
@@ -243,6 +243,32 @@ load _helpers
    [ "${actual}" = 'value2' ]
 }
 
+#--------------------------------------------------------------------
+# terminationGracePeriodSeconds
+
+@test "controller/Deployment: default terminationGracePeriodSeconds when revokeClientCacheOnUninstall is false by default" {
+  cd `chart_dir`
+  local actual=$(helm template \
+      -s templates/deployment.yaml  \
+      . | tee /dev/stderr |
+      yq '.spec.template.spec.terminationGracePeriodSeconds | select(documentIndex == 1)' | tee /dev/stderr)
+   [ "${actual}" = "120" ]
+}
+
+#--------------------------------------------------------------------
+# preDeleteHookTimeoutSeconds
+
+@test "controller/Deployment: default preDeleteHookTimeoutSeconds when revokeClientCacheOnUninstall is false by default" {
+  cd `chart_dir`
+  local object=$(helm template \
+      -s templates/deployment.yaml  \
+      . | tee /dev/stderr |
+      yq '.spec.template.spec.containers[0].args | select(documentIndex == 2)' | tee /dev/stderr)
+
+  local actual=$(echo "$object" | yq 'contains(["--pre-delete-hook-timeout-seconds=120"])' | tee /dev/stderr)
+  [ "${actual}" = "true" ]
+}
+
```

Comment:
Can we add a test for the revocation command arg that is added in this part of the template:
```
        {{- if .Values.controller.manager.clientCache.revokeClientCacheOnUninstall }}
        - --revoke-client-cache
        {{- end }}
```